composer.json

```diff
@@ -20,7 +20,7 @@
         "symfony/console": "~2.5",
         "symfony/process": "~2.5",
         "symfony/filesystem": "~2.5",
-        "henrikbjorn/lurker": "1.0.*@dev"
+        "henrikbjorn/lurker": "~1.1@dev"
```

Comment:
Right, it seems the repository is flint/lurker, but on packagist it is henrikbjorn/lurker. https://github.com/flint/Lurker So it is not a fork, simply the dependency version needs to be updated.